tools/helixer/helixer.xml

```diff
@@ -71,6 +80,20 @@
             <param argument="--peak-threshold" type="float" min="0" max="1" value="0.8" label="Peak threshold" help="This threshold specifies the minimum peak genic score required to accept the candidate region"/>
             <param argument="--min-coding-length" type="integer"  min="0" value="100" label="Minimum coding length"/>
         </section>
+
+        <param name="subsequence_length" type="text" label="subsequence-length parameter - If you do not want to keep the default value, please enter the new value." help="Default values are 21384 for fungi, 106920 for land plant, and 213840 for vertebrates and invertebrates"/>
```

Comment:
The label/help could be better: the labels from macros.xml were better, and the default values should be in help